### PR TITLE
add pod description to ovs-cni pod

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds pod description to ovs-cni pod

### CODE REVIEW

File: examples/ovs-cni.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ spec:
       labels:
         tier: node
         app: ovs-cni
+      annotations:
+        description: Ovs-cni pod configures Kubernetes networks on top of Open vSwitch bridges available on nodes
```
Comment (phoracek) [inline examples/ovs-cni.yml:19]: OVS CNI configures

We usually write it as OVS CNI.

And there is no reason to say that it is a pod

**Review by phoracek — CHANGES_REQUESTED** [review]

File: examples/ovs-cni.yml (hunk 1 above)
Comment (RamLavi) [reply examples/ovs-cni.yml]: Done

File: examples/ovs-cni.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@ spec:
       labels:
         tier: node
         app: ovs-cni
+      annotations:
+        description: OVS CNI configures Kubernetes networks on top of Open vSwitch bridges available on nodes
```
[the remark this answers is not in the document]
Comment (RamLavi) [reply examples/ovs-cni.yml]: ack. thanks!

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

Thanks a lot

**Pull request merged**